src/uwloc/data/main.py

```diff
@@ -0,0 +1,53 @@
+import glob
+import os
+
+import typer
+from typing_extensions import Annotated
+
+from .tile import MAX_HOURS, MAX_UNITS, create, init_tiledb, write_row
+from .wav import read_wav
+
+main = typer.Typer()
+
+
+def fail(msg: str) -> None:
+    typer.echo(msg)
+    raise typer.Exit(code=1)
+
+
+@main.command()
+def initdb(
+    dbpath: str,
+    max_units: Annotated[
+        int, typer.Argument(help="Maximum number of units (rows) to store in the array")
+    ] = MAX_UNITS,
+    max_hrs: Annotated[int, typer.Argument(help="Maximum amount of hours to store in the array")] = MAX_HOURS,
+) -> None:
+    typer.echo(f"Initializing TileDB in {dbpath}")
+    create(dbpath, max_units, max_hrs)
+
+
+@main.command(name="import")
+def _import(wavs_dir: str, dbpath: str) -> None:
+    typer.echo(f"Uploading WAV files from {wavs_dir} {dbpath}")
+    if not os.path.isdir(dbpath):
+        fail(f"TileDB '{dbpath}' not found. Run initdb command first.")
+
+    files = glob.iglob(os.path.join(wavs_dir, "**/*.[wW][aA][vV]"), recursive=True)
```

Comment:
I see that you just `os` library for system path manipulations. Are we assuming that this only runs on Unix machines only? If so this is fine, but if one tries to do Windows, this can be problematic. I suggest using `pathlib` to be more agnostic: https://docs.python.org/3/library/pathlib.html

Reply:
But `os.path.join` is OS independent, right? That's the advantage of using it vs hard-coding a separator (e.g. `path + "/" file`). What's the advantage of `pathlib` in this case?

Reply:
BTW, one bit of friction that `pathlib` introduces is that `Path` objects are not strings. E.g. TileDB chokes if you pass it a `Path` instead of a string for a directory location.

Reply:
Really `pathlib` has many additional features to make your code more readable, such as joining paths easily, built in glob features and navigating through directories, etc. My experience with the `os` library is that it has some quirks when you try to use it on a windows machine because it simply just manipulate string, rather than having a path object like the case of `pathlib`.

I remember my concern with using `os.path.join` in the first place! When you use it in context of creating path for database such as TileDB, id you're on windows you'll get the wrong path, since I assume TileDB probably uses unix style paths. For example say you want a data array path here: `/group/variable/myarray`. if you use `os.path.join` on windows you'll get `\group\variable\myarray`, which would be problematic. So in these cases, probably good to just do string joins.

I see the use of `pathlib` to simply get at the files, not so much creating the paths for TileDB. Basically the step where you pass the file object into wavinfo.